pythonFeed.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+vers=()
+
+echo "Initializing submodule..."
+git submodule update --init --recursive
+cd shared || exit && \
+git checkout feat/version-checking
+if [ -f automated-updates.sh ]; then
+  source automated-updates.sh
+else
+  echo "Check if submodule was loaded; automated-updates.sh is missing"
+  exit 1
+fi
+cd ..
+
+LATEST_VERSION=0
+VERSION_LIST=$(pyenv install --list | sed 's/^  //' | grep '^\d' | grep --invert-match 'dev\|a\|b')
+
+getPythonVersionLatest() {
+   
+    LATEST_VERSION=$(echo "$VERSION_LIST" | tail -1)
+}
+
+processLastVersion() {
+    LATEST_VERSION=$(echo "$LATEST_VERSION" | cut -f1,2 -d'.')
+    VERSION_LIST=$(echo "$VERSION_LIST" | grep -v "$LATEST_VERSION")
+    generateSearchTerms "PYTHON_VERSION" "$LATEST_VERSION"/Dockerfile
+      # shellcheck disable=SC2154
+    directoryCheck "$LATEST_VERSION" "$SEARCH_TERM"
+    if [[ $(echo $?) == 0 ]]; then
+        echo $LATEST_VERSION "is valid"
+        vars+=($LATEST_VERSION)
+    fi
+}
+
+while [ "${LATEST_VERSION}" != "3.5" ]
+do
+    getPythonVersionLatest
+    processLastVersion
+done
+
+VERSIONS=""
+for value in "${vars[@]}"
```

Comment:
each variable in the vars array is giving the major minor version. this needs to be the full version string

the syntax you have is correct with the vars[@]. try doing it without looping